app/src/common/chromium/com/igalia/wolvic/browser/api/impl/PanZoomCrontrollerImpl.java

```diff
@@ -1,9 +1,9 @@
 package com.igalia.wolvic.browser.api.impl;
 
 import android.view.MotionEvent;
+import android.view.ViewGroup;
 
 import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
 
 import com.igalia.wolvic.browser.api.WPanZoomController;
 
```

Comment:
Let's rename this class to PanZoomController (now it has a typo "PanZoomC**r**ontroller")